lib/pause.js

```diff
@@ -85,6 +91,10 @@ function parseInput(cmd) {
 
     const result = executeCommand instanceof Promise ? executeCommand : Promise.resolve(executeCommand);
     result.then((val) => {
+      if (isCustomCommand) {
+        console.log(val);
```

Comment:
could we use `output.print` here?

Reply:
Not really, this can be a very arbitrary output so I'd like to fully delegate that to NodeJS without any customization